chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -298,9 +298,9 @@ sync
 <% if @is_ses -%>
       <package>ses-release</package>
 <% else -%>
-  <% if @target_platform_version == "11.3" -%>
+  <% if @platform == "suse" && @target_platform_version == "11.3" -%>
```

Comment:
Not really, I guess. Although it could still be useful as reference if we want to add SP4 support.